documentation/config_file_puppetserver.md

```diff
@@ -90,6 +99,12 @@ http-client: {
 profiler: {
     enabled: true
 }
-~~~
+
+# Settings related to static catalogs.
+versioned-code: {
+    code-id-command: /opt/puppetlabs/server/apps/puppetserver/code-id-command_script.sh
+    code-content-command: /opt/puppetlabs/server/apps/puppetserver/code-content-command_script.sh
+}
+```
```

Comment:
Sounds good. Btw, can we mention that these scripts need to be executable by the user that puppetserver runs as?